docs/internal/dev_guidelines.rst

```diff
@@ -161,11 +161,11 @@ Whitespace
 * Namespace scopes that span most or all of a file do not cause indentation of their contents.
 * Otherwise lines ending in ``{`` indicate that subsequent lines should be indented one more level until ``}``.
 * Continuation lines (when a statement spans more than one line) can either be indented one more level, or indented to nest "under" an ``(`` or similar punctuation.
-* Add space around binary operators and after commas: ``a + b``
+* Add space around binary operators, between identical braces, and after commas: ``{ {}, {a + b} }``
 * Do not add space around parentheses for function calls, a properly formatted function call looks like ``func(arg1, arg2, arg3);``
 * Do not add space around brackets for indexing, instead it should look like: ``mymap[key]``
-* In general, do not add space :) Do not add extra spaces to make syntax elements (that span lines/statements) line up.
-  This only causes unnecessary changes in adjacent lines as the code evolves.
+* Do not add extra spaces to make syntax elements (that span lines/statements) line up. This only causes unnecessary changes in adjacent lines as the code evolves.
```

Comment:
```suggestion
* Do not add extra spaces to make syntax elements (that span lines/statements) line up.
  This only causes unnecessary changes in adjacent lines as the code evolves.
```

This was right, there should only be one sentence per line.

Reply:
Do we have guidelines for the formatting of the guidelines?

Reply:
Yes: https://opendds.readthedocs.io/en/master/internal/docs.html#rst-sphinx-usage

Reply:
Ironically, viewing them raw is the only way I tend to view them... but fine. We'll fix the typo and make it a single sentence.